enterprise_access/settings/base.py

```diff
@@ -505,11 +507,15 @@ def root(*path_fragments):
 SIMPLE_HISTORY_DATE_INDEX = False
 
 # Cache timeouts
-SUBSIDY_RECORD_CACHE_TIMEOUT = 60 * 5
-ENTERPRISE_USER_RECORD_CACHE_TIMEOUT = 60 * 5
-SUBSIDY_AGGREGATES_CACHE_TIMEOUT = 60 * 10
-
-ALL_ENTERPRISE_GROUP_MEMBERS_CACHE_TIMEOUT = 60 * 5
+DEFAULT_CACHE_TIMEOUT = 60 * 5  # 5 minutes
+CONTENT_METADATA_CACHE_TIMEOUT = 60 * 30  # 30 minutes
```

Comment:
[inform] Updating from 10 minutes to 30 minutes, for parity with the content metadata cache timeout in enterprise-subsidy ([source](https://github.com/openedx/enterprise-subsidy/blob/b1f95cf69613e9533b5d4e71ec87ba99941b8069/enterprise_subsidy/apps/content_metadata/api.py#L30)).